services/frontend/src/scenes/Profile/Profile.js

```diff
@@ -1,8 +1,30 @@
 import React from "react";
 import ProfileLayout from "./ProfileLayout";
 
+const sections  = [
```

Comment:
I wasn't 100% sure how to setup the components, figured data and functionality might want to be in the `.js` file instead of the layout so that the layout is only rendering the view. 

That said, this works as well so I moved it.